0131-bridging-gap-between-verification-mappers-and-anti-gaming-measures.md

```diff
@@ -11,47 +11,48 @@
-- Once a qualifing and/or accurate CDR is produced, the hotspot/radio will return to its original Oracle Hex Boost multiplier after 1 epoch.
-- Reasserting a hotspot/radio will reset the CDR, and another qualifing and/or accurate CDR will be required for the new location.
-- SPs can reduce the Oracle Hex Boosting multiplier of a hotspot or radios if they have reasonable evidence the POI hexes are being taken advantage of by a malicious actor (such as inaccurate an CDR).
-- Deployers will open a support ticket with the SP if they feel they have been mistakenly identified as malicious. This may involve, but not limited to, the SP attempting a physical verification of coverage at a given location, which would involve a connection to a radio (Wi-Fi or CBRS) to verify its existence. 
-- This HIP expires after 150 epochs (approximately 5 months) starting from formal HIP implementation. This gives the network time to implement a programmatic solution such as HIP-118.  However, the community can choose to extend this timeframe with a new vote if it feels sufficient progress has not been made on programmatic solutions.  It's suggested the community review the need for a vote several weeks ahead of expiration to provide time for deliberation and the general voting process.  Note that implementation of a programmatic solution would not reduce the 150 epoch duration.
+- Although a network operator's Hotspot may have its Oracle Hex Boost multiplier reduced, it can still earn rewards by way of data transfer.  Hotspots or radios are not banned or denied.
+- Once a qualifing and/or accurate CDR is produced, the Hotspot/radio will return to its original Oracle Hex Boost multiplier after 1 epoch.
+- Reasserting a Hotspot/radio will reset the CDR, and another qualifing and/or accurate CDR will be required for the new location.
+- SPs can reduce the Oracle Hex Boosting multiplier of a Hotspot or radios if they have reasonable evidence the POI hexes are being taken advantage of by a malicious actor (such as an inaccurate CDR).
+- Deployers can open a support ticket with the SP if they feel they have been mistakenly identified as malicious. This may involve, but not be limited to, the SP attempting a physical verification of coverage at a given location, which would involve a connection to a radio (Wi-Fi or CBRS) to verify its existence. 
+- This HIP expires after 150 epochs (approximately 5 months) starting from the community HIP implementation notification. This gives the network time to implement a programmatic solution such as HIP-118.  However, the community can choose to extend this timeframe with an additional HIP-131 extension vote if it feels sufficient progress has not been made on programmatic solutions.  It's suggested the community review the need for a vote several weeks ahead of expiration to provide time for deliberation and the general voting process.  Note that implementation of a programmatic solution would not reduce the 150 epoch duration.
```

Comment:
@madninja do you think this is long enough for verification mapping implementation?